{"name": "test_Login", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nBacktrace:\n\tOrdinal0 [0x00ED1ED3+2236115]\n\tOrdinal0 [0x00E692F1+1807089]\n\tOrdinal0 [0x00D766FD+812797]\n\tOrdinal0 [0x00DA55DF+1005023]\n\tOrdinal0 [0x00DA57CB+1005515]\n\tOrdinal0 [0x00DD7632+1209906]\n\tOrdinal0 [0x00DC1AD4+1120980]\n\tOrdinal0 [0x00DD59E2+1202658]\n\tOrdinal0 [0x00DC18A6+1120422]\n\tOrdinal0 [0x00D9A73D+960317]\n\tOrdinal0 [0x00D9B71F+964383]\n\tGetHandleVerifier [0x0117E7E2+2743074]\n\tGetHandleVerifier [0x011708D4+2685972]\n\tGetHandleVerifier [0x00F62BAA+532202]\n\tGetHandleVerifier [0x00F61990+527568]\n\tOrdinal0 [0x00E7080C+1837068]\n\tOrdinal0 [0x00E74CD8+1854680]\n\tOrdinal0 [0x00E74DC5+1854917]\n\tOrdinal0 [0x00E7ED64+1895780]\n\tBaseThreadInitThunk [0x7708FA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x77A77BBE+286]\n\tRtlGetAppContainerNamedObjectPath [0x77A77B8E+238]", "trace": "@allure.story('登入頁面輸入手機號碼')\n    @pytest.mark.run(order=4)\n    def test_Login():\n>           WebDriverWait(driver, 10).until(EC.presence_of_element_located((By.XPATH, \"/html/body/div[1]/div/div/main/div/div[2]/div/div/div[1]/div[2]/form/label/div/div[1]/div[2]/input\")))\n\nSSO_login_normal_test.py:65: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"25d0c5398d4d65005538286640d2cae8\")>, method = <function presence_of_element_located.<locals>._predicate at 0x000001A04BAC60E0>, message = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       Backtrace:\nE       \tOrdinal0 [0x00ED1ED3+2236115]\nE       \tOrdinal0 [0x00E692F1+1807089]\nE       \tOrdinal0 [0x00D766FD+812797]\nE       \tOrdinal0 [0x00DA55DF+1005023]\nE       \tOrdinal0 [0x00DA57CB+1005515]\nE       \tOrdinal0 [0x00DD7632+1209906]\nE       \tOrdinal0 [0x00DC1AD4+1120980]\nE       \tOrdinal0 [0x00DD59E2+1202658]\nE       \tOrdinal0 [0x00DC18A6+1120422]\nE       \tOrdinal0 [0x00D9A73D+960317]\nE       \tOrdinal0 [0x00D9B71F+964383]\nE       \tGetHandleVerifier [0x0117E7E2+2743074]\nE       \tGetHandleVerifier [0x011708D4+2685972]\nE       \tGetHandleVerifier [0x00F62BAA+532202]\nE       \tGetHandleVerifier [0x00F61990+527568]\nE       \tOrdinal0 [0x00E7080C+1837068]\nE       \tOrdinal0 [0x00E74CD8+1854680]\nE       \tOrdinal0 [0x00E74DC5+1854917]\nE       \tOrdinal0 [0x00E7ED64+1895780]\nE       \tBaseThreadInitThunk [0x7708FA29+25]\nE       \tRtlGetAppContainerNamedObjectPath [0x77A77BBE+286]\nE       \tRtlGetAppContainerNamedObjectPath [0x77A77B8E+238]\n\n..\\..\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:90: TimeoutException"}, "start": 1669879064789, "stop": 1669879074995, "uuid": "d6578c6a-289e-4e7a-8882-59f4c95fc987", "historyId": "73d2c4b7d957b7199d1976d63dbfc56f", "testCaseId": "9bd39bc6565d310565a94cbd2d824039", "fullName": "SSO_login_normal_test#test_Login", "labels": [{"name": "story", "value": "登入頁面輸入手機號碼"}, {"name": "tag", "value": "run(order=4)"}, {"name": "suite", "value": "SSO_login_normal_test"}, {"name": "host", "value": "C010700042"}, {"name": "thread", "value": "30460-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "SSO_login_normal_test"}]}